.gitignore

```diff
@@ -10,3 +10,4 @@ fathom
 
 assets/build
 assets/dist
+.idea
```

Comment:
This should be placed in your global `.gitignore` and not in the project's
```suggestion
```

Reply:
Fair enough, will remove